packages/lesswrong/components/posts/PostsItemTitle.jsx

```diff
@@ -10,7 +10,7 @@ const styles = theme => ({
   root: {
     color: "rgba(0,0,0,.87)",
     position: "relative",
-    lineHeight: "2rem",
+    lineHeight: "1.5rem",
```

Comment:
Changing this line-height broke the vertical-centering of the title text, causing all posts to be vertically offset from the center. We should likely ensure vertical centering with flexbox or the vertical-alignment property.

Reply:
If you mean the vertical offset in the continue-reading entries, that's deliberate (and it comes from a `top: -5` line in `hasResumeReading`/`& $title`, not from here). The offset is necessary to make room for the second line below. If you're referring to something else, I can't find it; everything looks correctly aligned to me, and changing the value of this `line-height` in the inspector doesn't cause anything to move.